priv/cabbage/apps/itest/lib/poller.ex

```diff
@@ -295,6 +291,10 @@ defmodule Itest.Poller do
         Process.sleep(@sleep_retry_sec)
         submit_typed(typed_data_signed, counter - 1)
 
+      %{"messages" => %{"code" => "operation:service_unavailable"}} ->
```

Comment:
hmmm. why does this happen and when? this particular response means there's something wrong on the client side. for example - alarms are raised.

Reply:
I removed it for now. It may be caused by non-deterministic nature of reorgs. I hope the error will disappear when using Reorg.execute_in_reorg function

Reply:
it didn't dissapear?

Reply:
no, I think it's related to Nginx and timeout limit to 10 seconds

Reply:
so as per slack, this actually is because we're raising alarms on our services - "stalled sync" when reorg is in progress.